### ISSUE TITLE
Pretrained weights

### ISSUE THREAD

**Issue by himkmk** [Lornatang/ESRGAN-PyTorch#28]
Hi, Lornatang.

First of all, thanks for your great work!

I just found a new issue on Jul 17 about the Adversarial loss term.
I wonder if the provided pretrained weights of RRDB/ESRGAN/Discriminator are...
1. trained weights from your previous code (the code with the error specified in the previous github issue)
2. or pretrained weights from the official github (possibly slightly modified to match you implementation of ESRGAN)

If  it is for the first case, are the results in the README.md file also wrong? 
To me, the error in the adversarial term seems to have a severe impact on the generator performance, since the generator would have been trained in the opposite way. (But, both the qualitative and quantitative results seemed quite similar to the original paper.)

**Comment by Lornatang** [comment]
The model weights is right.
The weight of this model was trained very early. At that time, it was trained with the correct code. The subsequent fixes were based on functional improvements of the code. Thank you for tell me some errors in the code.
Let's be better together!
